features/zones/Create.feature

```diff
@@ -110,15 +111,15 @@ Feature: zones / create
       """
       KUMA_SUBSCRIPTION_COUNT: 1
       """
-    When the URL "/zones+insights/test" responds with
+    Then the URL "/zones+insights/test" responds with
```

Comment:
**Question**: Why is this a `Then` instead of a `When`?

Reply:
I guess I read/wrote it like "when I click the create zone button, then the URL `/zones+insights/test` responds with.

Honestly, I don't mind either way as long as it reads fine. I can change it if you like?

Reply:
I read this as a test prerequisite and not an assertion so I think "When" was correct, but it’s not a big deal.

Reply:
I changed this test to read better overall, I used `Given`/`And`s for the arranging, and `Then`/`And`s for the assertions. I also moved some of the arranging to the start of the test. Will resolve, but feel free to reopen if necessary